mediaPlayerPath.txt

```diff
@@ -0,0 +1 @@
+C:\Program Files\VideoLAN\VLC\vlc.exe
```

Comment:
if it is in git ignore why it is here ? or am I confused

Reply:
so am i, trying to solve this one rn